src/core/ext/filters/client_channel/xds/xds_api.cc

```diff
@@ -1058,6 +1058,13 @@ grpc_error* RouteConfigParse(
     const envoy_api_v2_route_Route* route = routes[size - 1];
     const envoy_api_v2_route_RouteMatch* match =
         envoy_api_v2_route_Route_match(route);
+    const google_protobuf_BoolValue* case_sensitive =
```

Comment:
If xds_routing is not enabled, we shouldn't do this.  Instead, let's preserve the original behavior of ignoring this field.  (Note that if the prefix match is just "" or "/", then case_sensitive wouldn't have any effect anyway.)